src/plugins/console/public/lib/mappings/mappings.js

```diff
@@ -186,6 +191,10 @@ function loadTemplates(templatesObject = {}) {
   templates = Object.keys(templatesObject);
 }
 
+function loadIndexTemplates(data) {
+  indexTemplates = _.map(data.index_templates, 'name');
```

Comment:
Should be changed to:
```js
indexTemplates = (data.index_templates ?? []).map(({ name }) => name);
```

It will be painful to track the changes of all the string fields, used at `get`/`set`/`map`/etc, even after moving to TS, and it is better to avoid such a practice where it is possible. This code can be easily converted to vanilla js.